java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/message/ProxyReader.java

```diff
@@ -14,76 +14,116 @@
+ * The {@link SeRequest} optionally carries SE selection data and an APDU list.<br>
+ * The {@link SeResponse} contains the result of the selection and the responses to the APDUs.
  *
- * This interface should be implemented by any specific reader plugin.
+ * <p>The {@link SeRequest} are transmitted individually ({@link #transmitSeRequest(SeRequest,
+ * ChannelControl)} or by a list {@link #transmitSeRequests(List, MultiSeRequestProcessing,
+ * ChannelControl)} allowing applications to provide several selection patterns with various
+ * options.
+ *
+ * <p>{@link #releaseChannel()} is used to control the closing of logical and physical channels
+ * taking into account the observation mechanisms potentially in progress.
+ *
+ * @since 0.9
  */
 public interface ProxyReader extends SeReader {
 
   /**
-   * Transmits a Set of {@link SeRequest} (list of {@link SeRequest}) to a SE application and get
-   * back the corresponding a List of {@link SeResponse}.
-   *
-   * <p>The usage of this method is conditioned to the presence of a SE in the selected reader.
-   *
-   * <p>All the {@link SeRequest} are processed consecutively. The received {@link SeResponse} and
-   * placed in the List of {@link SeResponse}.
-   *
-   * <p>If the protocol flag set in the request match the current SE protocol and the
-   * keepChannelOpen flag is set to true, the transmit method returns immediately with a List of
-   * {@link SeResponse}. This response contains the received response from the matching SE in the
-   * last position of set. The previous one are set to null, the logical channel is open.
-   *
-   * <p>If the protocol flag set in the request match the current SE protocol and the
-   * keepChannelOpen flag is set to false, the transmission go on for the next {@link SeRequest}.
-   * The channel is left closed.
+   * Transmits the list of {@link SeRequest } and gets in return a list of {@link SeResponse}.
    *
-   * <p>This method could also fail in case of IO error or wrong card currentState &rarr; some
-   * reader’s exception (SE missing, IO error, wrong card currentState, timeout) have to be caught
-   * during the processing of the SE request transmission.
+   * <p>The actual processing of each {@link SeRequest} is similar to that performed by {@link
+   * #transmitSeRequest(SeRequest, ChannelControl)} (see this method for further explanation of how
+   * the process works).<br>
+   * If the multiSeRequestProcessing parameter equals to {@link
+   * MultiSeRequestProcessing#FIRST_MATCH}, the iteration over the {@link SeRequest} list is
+   * interrupted at the first processing that leads to an open logical channel state. In this case,
+   * the list of {@link SeResponse} may be shorter than the list of SeRequests provided as input.
+   * <br>
+   * If it equals to {@link MultiSeRequestProcessing#PROCESS_ALL}, all the {@link SeRequest} are
+   * processed and the logical channel is closed after each process.<br>
```

Comment:
For now, no. The logical channel is closed after each selection attempt.
This would imply not closing the channel before a new selection attempt.
So for example if we had a positive first selection, we select again without closing?

Reply:
Keep the current behavior